src/settings/JobProfiles/ViewJobProfile.js

```diff
@@ -402,17 +341,23 @@ export class ViewJobProfile extends Component {
       ? this.renderActionMenu
       : null;
 
-    return (
-      <Pane
+    const header = renderProps => (
```

Comment:
Move this function to the class methods and rename it, please.

Reply:
Done!